.travis.yml

```diff
@@ -40,9 +40,9 @@ after_success:
   - echo "NPM_TAG=$NPM_TAG"
   - >
     [ "${TRAVIS_BRANCH}" = "master" ] &&
-    npm version $GIT_TAG -m "Version $GIT_TAG built by Travis CI - https://travis-ci.com/$TRAVIS_REPO_SLUG/builds/$TRAVIS_JOB_ID" &&
+    yarn version --no-git-tag-version --new-version $GIT_TAG &&
```

Comment:
Seems like it's a good idea to make a new git tag? Tags are used by GitHub to track releases: https://github.com/meetup/meetup-web-platform/releases